### ISSUE TITLE
START /MIN WT?

### ISSUE THREAD

**Issue by vefatica** [microsoft/terminal#9053]
# Environment
Microsoft Windows 10 Pro for Workstations
10.0.19042.746 (2009, 20H2)
WindowsTerminalPreview_1.6.10272.0_x64
```none
Windows build number: [run `[Environment]::OSVersion` for powershell, or `ver` for cmd]
Windows Terminal version (if applicable):

Any other software?
```

# Steps to reproduce

In CMD.EXE, issue `START /MIN WT`

# Expected behavior
WT starts minimized and can be restored.

# Actual behavior
WT starts minimized and can't be restored.  The taskbar icon is rather lame.

I couldn't get good pics of the taskbar icon's popup thumbnails because they close when I activate the screen clipper.

**Comment by KalleOlaviNiemitalo** [comment]
Related issues, although not duplicates:

* <https://github.com/microsoft/terminal/issues/7374> a request for a `wt` command-line option that could be used instead of START /MIN.
* <https://github.com/microsoft/terminal/issues/7396> crash after START /MIN. As far as I know, it does not crash nowadays. This issue also mentions strange UI behaviour.

**Comment by zadjii-msft** [comment]
Yea, this is definitely unique compared to the other two. Repros easy enough. Real weird. I wonder if the OS is unhappy about us trying to set a window position when starting minimized is requested. Hard to know for sure, user32 is a mystery ¯\\\_(ツ)_/¯

**Comment by itsme-alan** [comment]
Screenshots from Alt-Tab:

![WT weirdness](https://user-images.githubusercontent.com/75578443/107468138-7ad0d800-6b8d-11eb-810e-d197721ea1f6.png)

**Comment by zadjii-msft** [comment]
We accidentally found the fix for this in 75d30651a, in #12979

**Comment by zadjii-msft** [comment]
Whoops, we reverted this but never reopened it.
